tests/conftest.py

```diff
@@ -3,15 +3,14 @@
 Init file for tests.
 """
 
-import json
-from typing import Any, Generator, Union
+from typing import Any
 
 import mongomock
 import pytest
-from django.http.response import HttpResponse
-from django.test import Client
 from pymongo import MongoClient
 
+from .client import APIClient
+
```

Comment:
should we be using relative imports in a small app like ours? there is another relative import in `tests/test_views/test_flags.py`.

Reply:
Oh sweet! I had not realized that we could use absolute imports in test_utils. I'll move the client module to test_utils and use an absolute import.